### PR TITLE
Using the parser to validate the values for global declarations , imports and struct fields

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx [hunk 1]
```diff
@@ -112,75 +111,22 @@ class PackageDeclarationNode extends React.Component {
...
-        const variableNode = NodeFactory.createVariable({ name: identifierNode,
-            initialExpression: literalNode,
-            typeNode,
-            const: isConst });
-        this.props.model.parent.addTopLevelNodes(variableNode);
+        value += ';\n';
+        parseContent(value)
+            .then((jsonTree) => {
+                if (jsonTree.topLevelNodes[0]) {
+                    this.props.model.parent.addTopLevelNodes(TreeBuilder.build(jsonTree.topLevelNodes[0]));
```
Comment (hevayo) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx]: Make sure you add variables before function and services.

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx [hunk 2]
```diff
@@ -191,15 +137,19 @@ class PackageDeclarationNode extends React.Component {
...
             return;
         }
-        // Create import node
-        const importNode = NodeFactory.createImport({ package: value });
-        this.props.model.parent.addTopLevelNodes(importNode);
+        value = 'import ' + value + ';\n';
+        parseContent(value)
+            .then((jsonTree) => {
+                if (jsonTree.topLevelNodes[0]) {
+                    this.props.model.parent.addTopLevelNodes(TreeBuilder.build(jsonTree.topLevelNodes[0]));
```
Comment (hevayo) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx]: Same as above make sure the order is proper.

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx (hunk 1 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx]: Fixed with commit 637385f

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx (hunk 2 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/package-declaration-node.jsx]: Fixed with commit 637385f

**Pull request merged**